src/main/java/org/kiegroup/kogibot/util/LabelsUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.kiegroup.kogibot.util;
+
+import java.io.IOException;
+import java.lang.invoke.MethodHandles;
+import java.util.List;
+
+import org.jboss.logging.Logger;
+import org.kiegroup.kogibot.util.Constants.DefaultLabels;
+import org.kohsuke.github.GHLabel;
+import org.kohsuke.github.GHPullRequest;
+import org.kohsuke.github.GHRepository;
+
+public class LabelsUtils {
+  private static final Logger LOG = Logger.getLogger(MethodHandles.lookup().lookupClass().getName());
+
+  public static final void addLabelsToPullRequest(List<String> labels, GHPullRequest pullRequest) throws IOException{
+    GHRepository ghRepository = pullRequest.getRepository();
+    List<GHLabel> ghLabels = ghRepository.listLabels().toList();
+    for (String labelName : labels) {
+      if (!ghLabels.stream().map(GHLabel::getName).anyMatch(l -> l.equals(labelName))) {
```

Comment:
not sure if it is a good idea, anyone would be able to add any aleatory label to the pr.
isn't better to enforce the label names we want?

Reply:
at least have the default ones like it was in the Labels class

Reply:
Well, some people may have different ways of dealing with PRs. The default ones are cool as a base but we should let people set their own.
For example, on operator, we have the BDD label. This would be too much work to have to update the bot each time you want a new label.
The configuration is from the target branch, so it is not that easy to change.
And I would not enforce the default ones neither. Let's make it configurable ;)